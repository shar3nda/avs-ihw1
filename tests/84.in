10
9 -2 0 9 -2 7 5 7 -2 10