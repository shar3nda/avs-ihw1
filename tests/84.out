-1 6 8 10 10 4 0 -1 1 -1 -1 -1